docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -219,6 +219,31 @@ _sql_db_isclean(){
     esac
 }
 
+_couch_db_isclean(){
+  # assume that we can connect to local CouchDB
+  # assume that env contains local couch user and password.
+  echo "$FUNCNAME: Checking if local couch db is empty"
```

Comment:
Perhaps add a detail in parenthesis like "(checking wmagent_summary db name)"?
In addition, I would suggest writing "CouchDB" in all these print statements that you added.

Reply:
oook, agree, done